api-dummy/public/schemas/libero/article/front.rng

```diff
@@ -1,26 +1,75 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 
-<grammar xmlns="http://relaxng.org/ns/structure/1.0" ns="http://libero.pub">
+<grammar ns="http://libero.pub" xmlns="http://relaxng.org/ns/structure/1.0"
+    xmlns:a="http://relaxng.org/ns/compatibility/annotations/1.0">
 
     <include href="../core.rng"/>
 
     <start>
+        <ref name="libero.front"/>
+    </start>
 
-        <element name="front">
-            <ref name="libero.front.content"/>
-        </element>
+    <div>
+        <a:documentation>front element</a:documentation>
 
-    </start>
+        <define name="libero.front">
+            <element name="front">
+                <ref name="libero.front.attributes"/>
+                <ref name="libero.front.content"/>
+            </element>
+        </define>
+
+        <define name="libero.front.attributes">
+            <ref name="libero.attrib.lang"/>
+        </define>
+
+        <define name="libero.front.content">
+            <interleave>
```

Comment:
This was missing, which meant a fixed ordering. This makes it flexible (since the order doesn't matter).